Resources/Prototypes/Entities/Mobs/Species/base.yml

```diff
@@ -256,6 +256,14 @@
         Heat: -0.07
       groups:
         Brute: -0.07
+  # DeltaV - damage on dragging downed people
+  - type: DamageOnDrag
+    damage:
+      groups:
+        Brute: 1
+    bleeding:
+      types:
+        Poison: 0.5
```

Comment:
this will let you stack poison forever by dragging a corpse around, there is no limit